WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1094,6 +1097,35 @@ private extension ProductFormViewController {
     }
 }
 
+// MARK: Action - Edit Product Downloads
+//
+private extension ProductFormViewController {
+    func showDownloads() {
+        guard let product = product as? EditableProductModel, product.isDownloadable()  else {
+            return
+        }
+
+        let downloadFileListViewController = ProductDownloadListViewController(product: product) { [weak self] data in
+            self?.onAddEditDownloadsCompletion(data: data)
+        }
+        navigationController?.pushViewController(downloadFileListViewController, animated: true)
+    }
+
+    func onAddEditDownloadsCompletion(data: ProductDownloadsEditableData) {
+        defer {
+            navigationController?.popViewController(animated: true)
+        }
+        let originalData = ProductDownloadsEditableData(productModel: product)
+        let hasChangedData = originalData != data
+        ServiceLocator.analytics.track(.productDownloadableFilesDoneButtonTapped, withProperties: ["has_changed_data": hasChangedData])
```

Comment:
nit: remove it for the moment, and add a `TODO`

Reply:
Changed